src/sentry/static/sentry/app/views/eventsV2/eventView.tsx

```diff
@@ -421,6 +422,49 @@ class EventView {
     });
   }
 
+  isEqualTo(other: EventView): boolean {
```

Comment:
Could this also be used in the existing code we have for determining if an API payload differs?